bin/convert-formula.sh

```diff
@@ -78,6 +83,15 @@ convert_formula() {
     sedi "/^\(  \)TEMPLATE\(:\)$/s//\1${NEW_NAME_PYSAFE}\2/" "$filename"
   done
 
+  # Where a section heading contains TEMPLATE, ensure the length of underlining
+  # matches the length of the new formula name
+  sedi "/TEMPLATE/{
+        n
+        s/========/$(repeat_char =)/
+        s/\^^^^^^^^/\\$(repeat_char ^)/"'
```

Comment:
Because of our reliance on [`m2r`](https://github.com/miyakogi/m2r#restrictions), the characters we use for `H1` through to `H6` are already defined.  So these two lines are effectively `H1` and `H3`.  While we don't have any `H2` that are affected, I believe it's forward-thinking to include that as well.

```suggestion
        s/========/$(repeat_char =)/
        s/\--------/\\$(repeat_char -)/
        s/\^^^^^^^^/\\$(repeat_char ^)/"'
```